content/blog/2018-05-02-onboarding-is-work.md

```diff
@@ -0,0 +1,325 @@
+--- 
+slug: onboarding-is-work
+title: 'How much work is onboarding?'
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+  - name: Noam Ross
+    url: http://twitter.com/noamross
+date: 2018-05-02
+preface: "This blog post is the second of a 3-post series about a data-driven overview of rOpenSci onboarding. Read the intro to the series [here](https://ropensci.org/blog/2018/04/26/satrday-ct-serie/) and the first post about data collection [here](https://ropensci.org/blog/2018/04/26/rectangling-onboarding/)."
+categories: blog
+topicid: 925
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- cloc
+output: 
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+Our [onboarding process](https://github.com/ropensci/onboarding/), that
+ensures that packages contributed by the community undergo a
+transparent, constructive, non adversarial and open review process,
+involves a lot of work from many actors: authors, reviewers and editors;
+but *how much work*? Managing the effort involved in the peer-review
+process is a major part of ensuring its sustainability and quality. In
+this post, we’ll take a look the effort put in by participants in the
+review process, and also learn something about exploring GitHub data
+along the way.
+
+### Work done by authors
+
+We can try to quantify the development work of authors by looking at the
```

Comment:
should you say "code" or "software" in first sentence to clarify what kind of development, or lines of what?